dockertest/subtest.py

```diff
@@ -154,27 +153,37 @@ def postprocess_iteration(self):
         """
         self.log_step_msg('postprocess_iteration')
 
-    @property
-    def control_config(self):
-        """
-        Represent operational control.ini's [Control] section as a dict or None
-        """
+    def _control_ini_section(self, section):
         if self._control_ini is None:
             fullpath = os.path.join(self.job.resultdir, 'control.ini')
             # Control-file & behavior cannot be assumed, file may not exist.
             try:
-                self._control_ini = config.ConfigDict('Control')
+                self._control_ini = config.ConfigDict(section)
                 self._control_ini.read(open(fullpath, 'rb'))
             except (IOError, OSError, Error):
                 self.logwarning("Failed to load reference '%s' and/or"
-                                "it's '[Control]' section.", fullpath)
+                                "it's '[%s]' section.", fullpath, section)
```

Comment:
As long as you're touching this line, could you s/it's/ its/? (remove apostrophe, add leading space to separate from preceding and/or) -- thanks